Project.toml

```diff
@@ -45,7 +45,7 @@ ApproxManifoldProducts = "^0.1"
 Combinatorics = "^0.7, 0.8, 0.9, 1"
 CoordinateTransformations = "0.5, 0.6"
 DataStructures = "0.15, 0.16, 0.17, 0.18, 0.18, 0.19, 0.20"
-DistributedFactorGraphs = "0.8, 0.9"
+DistributedFactorGraphs = "0.10"
```

Comment:
To keep continuity we should try keep DFG = "0.9, 0.10" otherwise it creates real difficulty moving a system forward -- overly narrow compat creates islands of compatibility that hard to navigate, especially since Caesar.jl has 120 dependencies.